### ISSUE TITLE
Python compatibility

### ISSUE THREAD

**Issue by LeoAlioth** [Burtond/Electrodacus#8]
install script does not run on python 3 because of missing parenthesis on print functions

**Comment by Burtond** [comment]
This has been corrected.  Thank you for letting me know.

**Comment by Burtond** [comment]
Resolved

**Comment by LeoAlioth** [comment]
First thank you for providing this code to set up things like this. I have found some other things I had to do in SerialToInfluxDB to make it work on python 3.X.

101    sign = var_sbms[28]
should be 
101    sign = chr(var_sbms[28])
otherwise all the battery currents are logged as positive

and there is also something weird going on with parsing ADC data from the compressed string, 

First is that you area always assigning values to ADC2 variable, and the second one is that after fixing that, (and dividing values by 1000 to get volts instead of mV), Voltage from what shows on sbms0 as ADC2 gets parsed as ADC4. (i don't know what reads into ADC2 and ADC3 as I don't have anything else connected. Values read 0, and haven't yet put more time into it to figure it out).

Also it might be good to update grafana version in the install script.
